lib/index.integration.spec.js

```diff
@@ -0,0 +1,74 @@
+const fs = require('fs');
```

Comment:
Can we remove this file for now - and we will come back to add an `index.spec.js`

Reply:
file has been removed